src/components/OnboardingMenu/components/OnboardingMenuItem/frequentFunctions/OnboardingMenuItemText/OnboardingMenuItemText.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import {block} from '../../../../../utils/cn';
+
+import './OnboardingMenutemText.scss';
+
+const cnOnboardingMenuItemText = block('guide-menu-item-text');
+
+export type OnboardingMenuItemTextProps = {
+    text: string;
+};
+
+const OnboardingMenuItemText = ({text}: OnboardingMenuItemTextProps) => (
+    <span className={cnOnboardingMenuItemText()}>{text}</span>
+);
+
+export const addText = (text: string) => <OnboardingMenuItemText text={text} />;
```

Comment:
Why we need this helper? `OnboardingMenuItemText` is already act like a function of `text`

Reply:
it seemed to me that it looks more convenient than using the component itself.

`addText('Test text')`
```
{[
    addImage(
        'https://storage.yandexcloud.net/uikit-storybook-assets/changelog-dialog-picture-1.png',
        '2019',
    ),
    addText('Test text')
]}
```

or

`<OnboardingMenuItemText text={'Test text'}/>`
```
{[
    addImage(
        'https://storage.yandexcloud.net/uikit-storybook-assets/changelog-dialog-picture-1.png',
        '2019',
    ),
    <OnboardingMenuItemText text={'Test text'}/>
]}
```

Reply:
Feels unnatural to use React like that